### PR TITLE
Avatar and Badge 

### PR DESCRIPTION
**Pull request by SatwikaKacham**

### CODE REVIEW

File: badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+.badge-online{
+    background-color: green;
+}
+.badge-offline{
+    background-color: #f9a61a;
+}
+.badge-dnd{
+    background-color: #f34648;
+}
```
Comment (kart-c) [inline badge/badge.css]: instead of hard-coding colors you can use variables

File: avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+.avatar{
+    height: 5rem;
+    width:5rem;
+    border-radius: 5rem;
+    font-size: 3rem;
+    font-weight: 600;
+    padding:0.7rem;
+    color:rgb(49, 49, 49);
+    /* color:rgb(231, 231, 231); */
```
Comment (kart-c) [inline avatar/avatar.css]: you can remove this comment if it is not required

**Review by kart-c — CHANGES_REQUESTED** [review]

**Review by kart-c — APPROVED** [review]
Approved

**Review by mohammed-sas — APPROVED** [review]
you are good to merge

**Pull request merged**